examples/ERP/plot_classify_EEG_quantum.py

```diff
@@ -0,0 +1,127 @@
+"""
+====================================================================
+ERP EEG decoding with Quantum Classifier.
+====================================================================
+
+Decoding applied to EEG data in sensor space decomposed using Xdawn.
+After spatial filtering, covariances matrices are estimated, then projected in
+the tangent space and classified with a quantum classifier
+
+"""
+# Author: Gregoire Cattan
+# Modified from plot_classify_EEG_tangentspace.py
+# License: BSD (3-clause)
+
+import numpy as np
+
+from pyriemann.estimation import XdawnCovariances
+from pyriemann.tangentspace import TangentSpace
+from pyriemann_qiskit.classification import QuanticSVM
+
+import mne
+from mne import io
+from mne.datasets import sample
+
+from sklearn.pipeline import make_pipeline
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import confusion_matrix, ConfusionMatrixDisplay
+
+from matplotlib import pyplot as plt
+
+# cvxpy is not correctly imported due to wheel not building
+# in the doc pipeline
+__cvxpy__ = True
+try:
+    import cvxpy
```

Comment:
Why do you need this import, since the example does not use any functions/classes from this package?

Reply:
Oh sorry, I made some comments in the initial PR I forgot to report here. 
Wheel fail to build the cvxpy library in the doc pipeline. Therefore it happens that the cvxpy library is not imported and that Qiskit is not working when building the doc. 
This is a temporal workaround to avoid the pipeline to fail, as explained in the comment above.

Reply:
Ok. IMO, an issue can be opened later to properly solve this problem.

Reply:
Thank you for your answer. I confirm issue #5 was created.